README.rst

```diff
@@ -20,33 +20,68 @@ configurations remain synchronized with the state of all databases in the
 cluster. PgQ is used for buffering mutation records until they are collected by
 the relay and forwarded to their final destination.
 
+Function
```

Comment:
I was trying to find a word that says, "Here is what this thing does at the layer beyond just the conceptual level."  "Overview" seems okay too..not sure.  How about "Usage?"